packages/sui-studio/bin/sui-studio-generate.js

```diff
@@ -154,7 +154,7 @@ const defaultContext = `module.exports = {
 
 const prepare = swc
   ? 'sui-js-compiler'
-  : 'babel --presets sui ./src --out-dir ./lib'
+  : 'npm run build:js && npm run build:styles'
```

Comment:
I think that the problem is different, we should only change js compilation if swc option is true
It means:
```
const buildJs = swc
  ? 'sui-js-compiler'
  : 'babel --presets sui ./src --out-dir ./lib'
```
And then:
```
"scripts": {
    "prepare": "npm run build:js && npm run build:styles",
    "build:js": "${buildJs}",
    "build:styles": "cpx './src/**/*.scss' ./lib"
  },
```

Isn't it? @midudev

Reply:
Changed!